src/test/java/com/library/controller/AuthorControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.library.controller;
+
+import com.library.model.dto.AuthorDto;
+import com.library.service.implementation.AuthorServiceImpl;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class AuthorControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private AuthorServiceImpl authorService;
+
+    private static final Long ID = 1L;
```

Comment:
It is not so necessary, but use IDE generation for this ID.

Reply:
Is it plugin?

Reply:
As i know it is IDE standard behavior.